types/testingutils/keymanager.go

```diff
@@ -25,11 +27,33 @@ type testingKeyManager struct {
 	slashableDataRoots [][]byte
 }
 
+var (
+	instancesMap = make(map[string]*testingKeyManager)
+	mu           sync.Mutex
+)
+
+func getHash(data [][]byte) string {
+	hasher := sha256.New()
```

Comment:
you can consider using MuHash or something else which is faster

Reply:
fnv is easy fast and simple for tasks like this. built in the go stdlib